javascript/collapsible.js

```diff
@@ -19,9 +19,88 @@ function collapseAllGroups() {
   }
 }
 
+var strclass = '';
+var res = '';
+$("tbody > tr").each(function() {
+
+    if ($(this).attr("title") == undefined) {
+        return;
+    }
+
+    if (strclass == '') {
+        strclass = $(this).attr("title");
+        res = strclass.split(" ");
+        return;
+    }
+
+    let strclasscheck = $(this).attr("title");
+    let rescheck = strclasscheck.split(" ");
+
+    if (rescheck.includes(res[res.length - 1])) {
+        $(this).attr("style","visibility: collapse");
+    }
+    else {
+        strclass = $(this).attr("title");
+        res = strclass.split(" ");
+    }
+
+});
+
 $("table.collapsible thead").find("th").on("click", function() {
     $(this).get(0).className = ($(this).get(0).className == 'open') ? 'closed' : 'open';
     $(this).closest("table").find("tbody").toggle();
-    
 });
 
+$("tr").find("td.path").on("click",function(event) {
+
+    event.stopPropagation();
+
+    let $target = $(event.target);
+    let strclass = $target.closest("tr").attr("title");
+    let res = strclass.split(" ");
+
+    if ($target.closest("tr").next().attr("title") == undefined) {
+        return;
+    }
+
+    let strclasscheck = $target.closest("tr").next().attr("title");
+    let rescheck = strclasscheck.split(" ");
+    let performaction = $target.closest("tr").attr("customaction") == 'close'? 'open' : 'close';
```

Comment:
All of the above names read badly again and should be camel-cased:

    let strClassCheck =
    let resCheck =
    let performAction =

Reply:
Changed the variable name to camel-cased. For the handlers.
Also spaced out the entire Js file to try to make it consistent.